app/src/main/java/edu/grinnell/kdic/MainActivity.java

```diff
@@ -374,13 +380,13 @@ public void showVisualizeFragment() {
             mPlaybackToolbar.setNavigationIcon(R.drawable.ic_keyboard_arrow_down_white_24dp);
 
             // move the play button to the middle
-            final float shiftAmnt = (mPlaybackToolbar.getWidth() - playPauseButton.getWidth()) / 2;
+            final float shiftAmnt = (mPlaybackToolbar.getWidth() - mPlayPauseButton.getWidth()) / 2;
```

Comment:
Why are we dividing by some random magic number? What is 2 here? Extract as a constant and name properly